### PR TITLE
Add persistency

### PR DESCRIPTION
**Pull request by mobilutz**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ gem 'rails', '4.2.6'
 gem 'bootstrap-sass', '~> 3.3.5'
 gem 'jbuilder', '~> 2.0'
 gem 'jquery-rails'
-gem 'mongoid', '~> 4.0.2'
+gem 'mongoid'
```
Comment (coding-bunny) [inline Gemfile]: wow, no version?

Comment (coding-bunny) [inline Gemfile]: wow, no version?

Comment (mobilutz) [inline Gemfile]: You don't always need a version in the Gemfile ;)

File: app/controllers/developers_controller.rb [hunk 2]
```diff
@@ -62,11 +62,18 @@ def destroy
     end
   end
 
+  # GET /developers/generate_assignment
+  def generate_assignment
+    all_devs = Developer.order_by(name: 'asc').to_a
+    MentorshipAssigner.new(all_devs).perform
```
Comment (ujh) [inline app/controllers/developers_controller.rb]: Does this line save stuff to the DB? If so maybe it would make sense to switch that page to `POST`.

**Pull request merged**